.github/workflows/ferretdb_go_tests.yml

```diff
@@ -26,10 +26,7 @@ env:
 jobs:
   test:
     name: Test
-    # https://www.ubicloud.com/docs/about/pricing#github-actions
-    # https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates-for-larger-runners
-    runs-on: ubicloud-standard-4
-
+    runs-on: ubuntu-24.04
```

Comment:
GitHub runners for public repos are free and provide 4 vCores now too. And they are slightly faster (and free):
* https://github.com/FerretDB/documentdb/actions/runs/13435638088/job/37537160391?pr=7
* https://github.com/FerretDB/documentdb/actions/runs/13436942232/job/37541413047?pr=7

Reply:
Free is good!